Add-ons/UmbracoForms/Editor/Attaching-Workflows/index.md

```diff
@@ -39,7 +43,7 @@ A new workflow can be of different types and Umbraco Forms ships with a few defa
 
 ### Update Type-specific Settings
 
-Once the Workflow Type has been selected, you will need to configure the workflow. There are various settings depending on the type that has been selected.
+Once the Workflow Type has been selected, you will need to configure the workflow. There are settings depending on the type that has been selected.
```

Comment:
I think that removing "various" in this case actually takes away some of the meaning. So how about adding "different" instead?

```suggestion
Once the Workflow Type has been selected, you will need to configure the workflow. There are different settings depending on the type that has been selected.
```